charts/jenkins/jobs/nuxeo/lts/release-jira.groovy.gotmpl

```diff
@@ -25,13 +25,18 @@ pipelineJob('nuxeo/lts/2023/release-jira-2023') {
           }
         }
       }
-      // Lightweight checkout
-      lightweight(true)
+      // no lightweight checkout in order to make branch parameter work
+      lightweight(false)
       // Script Path
       scriptPath('Jenkinsfiles/release-jira.groovy')
     }
   }
   parameters {
+    string {
+      name('NUXEO_BRANCH')
+      description('Branch to release in Jira.')
```

Comment:
Branch to release (simple and consistent with the other jobs if it's the same everywhere)